tools/icqsol_translate_shape/icqsol_translate_shape.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="icqsol_translate_shape" name="translate shape" version="1.0.0">
+    <description></description>
+    <macros>
+        <import>icqsol_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command>
+<![CDATA[
+    python $__tool_directory__/icqsol_translate_shape.py
+    --input "$input"
+    --displacement_x $displacement_x
```

Comment:
You're going to need these here:
    --input_file_format_and_type $input.ext
    --input_dataset_type $input.metadata.dataset_type